tests/Unit/ConfigTest.php

```diff
@@ -21,13 +21,6 @@ class ConfigTest extends TestCase
     protected function getEnvironmentSetUp($app): void
     {
         $app['config']->set('graphql', [
-            'prefix' => 'graphql_test',
-
-            'routes' => [
-                'query' => 'query/{graphql_schema?}',
-                'mutation' => 'mutation/{graphql_schema?}',
-            ],
```

Comment:
This feature is not supported anymore.

The `prefix` is already removed, because it was otherwise not used in any of the remaining tests.